frontend-api/src/test/java/uk/gov/di/authentication/frontendapi/lambda/VerifyCodeHandlerTest.java

```diff
@@ -679,6 +690,46 @@ void shouldReturn204ForValidResetPasswordRequestUsingTestClient() {
         assertThat(result, hasStatus(204));
     }
 
+    @ParameterizedTest
+    @MethodSource("codeRequestTypes")
+    void shouldDeleteCountOnSuccessfulSMSCodeRequest(
+            CodeRequestType codeRequestType, JourneyType journeyType) {
+        when(codeStorageService.getOtpCode(EMAIL, MFA_SMS)).thenReturn(Optional.of(CODE));
+        when(configurationService.isAuthenticationAttemptsServiceEnabled()).thenReturn(true);
+
+        var result = makeCallWithCode(CODE, MFA_SMS.toString(), journeyType);
+
+        verify(authenticationAttemptsService, times(1))
```

Comment:
Yeah here I was suspicious that although we're passing in journey type as a param, we're expecting this to be deleted in all cases. Had a look at the implementation and think we're missing a check